code/style.css

```diff
@@ -5,18 +5,102 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  /* background-image: url(./assets/background.jpeg); */
+  background-color: #585858;
+  font-family: 'Palette Mosaic', cursive;
 }
 
 h1 {
   font-weight: bold;
   font-size: 44px;
   line-height: 34px;
-  color: #fff;
+  color: rgb(243, 243, 243);
   text-align: center;
 }
 
+.title {
+  background-color: #585858;
+  padding:10px;
+
+
+}
+
+ul{
+  display:flex;
+  justify-content: center;
+}
+
+
+ul li{
+  list-style:none;
+  font-family: 'Palette Mosaic', cursive;
+  font-size:5em;
+  letter-spacing:15px;
+  text-shadow:none;
+  animation:animate 1s infinite linear;
+}
+
+@keyframes animate{
```

Comment:
This is really well done with the H1! Had no idea that you could do it like this.

Reply:
Found on google, there are a lot of great animation you can do with css!